### PR TITLE
[indigo] Add Docker to main MoveIt! repo

### PR DESCRIPTION
**Pull request by davetcoleman**
Almost same as https://github.com/ros-planning/moveit/pull/53 and jade version, but doesn't use shadow fixed or call rosdep update

Tested locally

### CODE REVIEW

**Comment by rhaschke** [comment]
You should adapt link for "Docker Containers" in [README.md](https://github.com/ros-planning/moveit/blob/indigo-devel/README.md) to https://github.com/ros-planning/moveit/tree/indigo-devel/.docker

**Comment by davetcoleman** [comment]
I think better would be the new page on the website I made - can you review and merge it?

https://github.com/ros-planning/moveit.ros.org/pull/36

**Comment by davetcoleman** [comment]
Docs have been merged, I updated the link, this is ready

File: .docker/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+[![Docker Pulls](https://img.shields.io/docker/pulls/moveit/moveit.svg?maxAge=2592000)](https://hub.docker.com/r/moveit/moveit/)
+[![Docker Stars](https://img.shields.io/docker/stars/moveit/moveit.svg)](https://registry.hub.docker.com/moveit/moveit/)
+
+## Available Images
+
+For each ROS distribution there are 3 images, built on top of a standard [osrf/ros:indigo-desktop](https://github.com/osrf/docker_images/blob/master/ros/indigo/indigo-desktop/Dockerfile) (or other distro version) image:
+
+ - **source image**: contains all dependencies and a full MoveIt! workspace downloaded and built to ~/ws_moveit/src
+ - **release image**: the full debian-based install of MoveIt! using apt-get
+ - **ci image**: an image optimized for running continuous integration with Travis and [moveit_ci](https://github.com/ros-planning/moveit_ci)
```
Comment (jrgnicho) [inline .docker/README.md]: Shouldn't this file also have a link to [http://moveit.ros.org/install/docker/](http://moveit.ros.org/install/docker/) ?

Comment (davetcoleman) [inline .docker/README.md]: Sure - that page didn't exist when I made this README :)

Comment (davetcoleman) [inline .docker/README.md]: done

**Comment by davetcoleman** [comment]
Ready to merge

**Comment by rhaschke** [comment]
@davetcoleman 
Could you unify the README.md's (using the links http://moveit.ros.org/install/docker) in Jade and Kinetic too?

**Comment by davetcoleman** [comment]
Unified (cherry-picked) for J/K

**Pull request merged**